docs/manual-CN/server-administration/storage-metrics.md

```diff
@@ -0,0 +1,73 @@
+# Nebula  Storage Metrics
+## 介绍
+
+目前，Nebula 支持通过 HTTP 方式来获取 Storage Service 层操作的一些基本性能指标。
+
+每一个性能指标都由三部分组成，分别为指标名，统计类型，时间范围。<br />`<counter_name>.<statistic_type>.<time_range>`<br />下面将分别介绍这三部分。
+
+### 指标名
+每个指标名都由接口名加指标名构成，目前支持获取如下接口
+
+```
+获取一个点周边节点的 id 和属性，以及相关边的属性 get_bound
```

Comment:
mark this line (get_bound) as intenal use only

put command as first col, and explaination as second column